docs/components/power-sensor/_index.md

```diff
@@ -238,6 +238,58 @@ power, err := myPowerSensor.Power(context.Background(), nil)
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetReadings
+
+Get the measurements or readings that this power sensor provides.
+If a sensor is not configured to have a measurement or fails to read a piece of data, it will not appear in the readings dictionary.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` [(Optional\[Dict\[str, Any\]\])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional\[float\])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- [(Mapping\[str, Any\])](https://docs.python.org/3/library/typing.html#typing.Mapping): The measurements or readings that this sensor provides.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/power_sensor/client/index.html#viam.components.power_sensor.client.PowerSensorClient.get_readings).
+
+```python
+my_sensor = Sensor.from_robot(robot=robot, name='my_sensor')
```

Comment:
I think this should be `PowerSensor.from_robot`

Reply:
Ahh thank you! Fixed